src/main/java/org/jenkinsci/test/acceptance/junit/JenkinsAcceptanceTestRule.java

```diff
@@ -62,15 +60,6 @@ public void evaluate() throws Throwable {
                     System.out.printf("Skipping %s%n", description.getDisplayName());
                     e.printStackTrace();
                     throw e;
-                } catch (Exception|AssertionError e) { // Errors and failures
```

Comment:
After moving this logic out of this method into the failed method in the DiagnosticRule class - will screenshots still be taken on Assertion Errors? Or, only on Exceptions? I guess this is determined by what conditions causes the failed method to be invoked.

Reply:
Just checked to be sure, the failed() method is called for both exceptions and AssertionError.

Reply:
@romainverduci awesome